tests/antithesis/test-template/robustness/main.go

```diff
@@ -118,9 +120,9 @@ func testRobustness(ctx context.Context, hosts, persistedRequestdirs []string, r
 	} else {
 		assert.Always(result.Linearization.Linearizable == porcupine.Ok, "Linearization validation passes", nil)
 	}
-	err = result.Linearization.Visualize(lg, reportPath)
-	if err != nil {
-		lg.Error("Failed to save visualization", zap.Error(err))
+	r.Visualize = result.Linearization.Visualize
+	if err := r.Report(true, false, true, reportPath); err != nil {
```

Comment:
Could not write a comment on line 62, oh GitHub.

Think the report path no longer makes sense
```
	defaultReportPath = "/var/report/history.html"
	localReportPath   = "history.html"
```
We should modify it to make sense both outside and inside.

```
	defaultReportPath = "/var/report/"
	localReportPath   = "report"
```

For container this will dump report in mounted directory, and for local run it will create a `report` directory.
The only one problem I see is conflicts when report is written on directory that already contains it. This mangles the data. Let's leave the cleanup up to the `make antithesis-clean` for now.

Reply:
I have added a step to `init` to clean up before `chown` the folder the user.  This should help with the clean up.